src/data/faq.json

```diff
@@ -446,6 +446,18 @@
                             "<ol><li>Go to your device settings</li><li>Select 'Security & Location' -> 'Location' -> 'App level permissions'</li></ol>",
                             "Here you can deny other apps permission to use your location if you don't think they need it. Since Corona-Warn-App doesn't need this permission, it doesn't appear in the list."
                         ]
+                    },
+                    {
+                        "title": "[Google/Android] A card on my Start Screen is notifying me that my device is not or only partially compatible. What does this mean?",
+                        "anchor": "incompatibility_warning",
+                        "active": true,
+                        "textblock": [
+                            "If you see such a notice, your device is not able to send any random IDs via Bluetooth. This can be caused by a restriction due to your combination of hardware and software. Because of it, the Corona-Warn-App cannot use Bluetooth Low Energy in its so-called “peripheral mode”.",
+                            "In case you are infected and you decide to warn others, only users who were checked in to the same event or the same location, at the same time as you were, can be warned.",

```

Comment:
This is very similar to the last sentence. I suggest unifying the information.

We have to be very careful here to not scare away users. So it should be very clear that
* this mostly affects warning others, not receiving warnings
* it only affects the proximity-based ID exchange. Other functionalities of the app (contact journal, etc.) are not affected
* Warning other via Check-In is still possible (this is already in the entry)

Reply:
@heinezen I think it would be easier to explain if we split this up into two FAQ entries – one for "partial incompatibility" and one for "full incompatibility" (which is very rare anyway, as I don't know any devices without Bluetooth chips). Do you think that would be a good idea, and specifically, whether there would be any chance we could still get a change that directs users to the correct entry into the 2.2 release?

Reply:
@fynngodau I don't think it is strictly necessary if the full incompatibility is so rare. One problem I see in https://github.com/corona-warn-app/cwa-app-android/pull/2873 is that it's not clear from the error message what type of incompatibility (full or partial) the user is facing. This can be a problem because it might leave the user more confused after reading the FAQ article than after reading the error messages.

IMO we have two user-friendly options:
1. Make two distinct FAQ articles like you suggested, one for partial and one for full incompatibility. Some of the information would have to be in both articles, but that's not a problem. The app then needs to link to the correct entry.
2. Make two distinct error headlines in the app, one for partial and one for full incompatibility. We can then link to the same FAQ entry and include separate sections for each error type. This would make the in-app information slightly more useful and the FAQ entry contains information for both cases which *can* be better if people just do an internet search for the error (in option 1, they might see the wrong entry).

Option 1 is probably faster, so I would go for that.

> whether there would be any chance we could still get a change that directs users to the correct entry into the 2.2 release?

We can try because it's not a big change, but I don't know if we'll get approval soon enough. However, it's worth a shot I think. It's best if you make a 2nd PR for the two separate entries and we'll leave this one open just in case we don't have enough time. But let''s hope for the best :)

Reply:
@heinezen I opened an Android PR and am hoping to receive feedback quickly: https://github.com/corona-warn-app/cwa-app-android/pull/3101